paparazzi-junit4/src/test/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtensionTest.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("invisible_reference", "invisible_member")
```

Comment:
allows "friend references", i.e., extending internal access across cross project-modules.  By default, kotlin only allows code within the same module to access `internal` members.

Reply:
Not K2 friendly: https://youtrack.jetbrains.com/issue/KT-67920

Maybe it is possible to befriend another module's sourceSet with this?
https://github.com/TWiStErRob/repros/blob/main/kotlin/test-fixtures-and-suites-friends/build.gradle.kts